packages/EasyCore/src/Bridge/Laravel/Middleware/TrimStrings.php

```diff
@@ -57,8 +57,11 @@ private function clean(Request $request): void
         }
     }
 
+    /**
+     * @param ParameterBag<string, mixed> $bag
+     */
     private function cleanParameterBag(ParameterBag $bag): void
     {
-        $bag->replace($this->cleaner->clean($bag->all(), $this->except));
+        $bag->replace($this->cleaner->trim($bag->all(), $this->except));
```

Comment:
Please, rename property from `cleaner` to `trimmer`

Reply:
Done.